core/app/models/spree/app_configuration.rb

```diff
@@ -248,6 +248,14 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean] Whether use of an address in checkout marks it as user's default
     preference :automatic_default_address, :boolean, default: true
 
+    # @!attribute [rw] shipping_rate_sorter_class
+    #   @return [Class] The class used to sort shipping rates via the Estimator
+    preference :shipping_rate_sorter_class, :class, default: Spree::Stock::ShippingRateSorter
+
+    # @!attribute [rw] shipping_rate_selector_class
+    #   @return [Class] The class used to choose shipping rates via the Estimator
+    preference :shipping_rate_selector_class, :class, default: Spree::Stock::ShippingRateSelector
```

Comment:
We can't store classes as preferences this way (there's no understanding of `:class` as a type). But this could be done like `variant_search_class` or `promotion_chooser_class` below